parm/parm_gdas/aero_jedi_fix.yaml

```diff
@@ -7,5 +7,5 @@ copy:
   - !ENV ${DATA}/fv3jedi/fmsmpp.nml
 - - !ENV ${FV3JEDI_FIX}/fv3jedi/fv3files/field_table_gfdl
   - !ENV ${DATA}/fv3jedi/field_table
-- - !ENV ${FV3JEDI_FIX}/fv3jedi/fieldmetadata/gfs-aerosol.yaml
-  - !ENV ${DATA}/fv3jedi/gfs-restart.yaml
+- - !ENV $(HOMEgfs)/sorc/gdas.cd/parm/io/fv3jedi_fieldmetadata_restart.yaml
```

Comment:
I'm guessing @aerorahul will want this to be instead linked at the link_workflow.sh stage and put somewhere else. Question is is that an 'eventually' thing or a 'now' thing?

Reply:
Correct.  This should be linked to `${HOMEgfs}/parm/gdas/io/fv3jedi_fieldmetadata_restart.yaml` in `sorc/link_workflow.sh` at the time the workflow is being setup for vertical structure.

From this location, it can be copied to `DATA` at job runtime.

Reply:
Per offline convo, I will do this, just not in this PR.